autoendpoint/src/metrics.rs

```diff
@@ -128,28 +128,25 @@ impl Metrics {
     }
 
     // increment a counter with no tags data.
-    pub fn incr(self, label: &str) {
-        self.incr_with_tags(label, None)
+    pub fn incr(self, metric: MetricName, tags: Option<Tags>) {
```

Comment:
I'm confused.
`incr()` was pretty much a convenience method to allow the caller not to include the tags (we would just pass a `None` to `incr_with_tags`), yet now they have to include the `None`?
Doesn't including the tags make this less convenient?

Reply:
Thanks for catching this! 👍